### PR TITLE
Enable the `defaultZoomDelay` option/preference unconditionally, and other (small) improvements

### PR DESCRIPTION
**Pull request by Snuffleupagus**
*Please refer to the individual commit messages for additional details.*

### CODE REVIEW

File: web/pdf_viewer.js [hunk 1]
```diff
@@ -230,19 +230,15 @@ class PDFViewer {
...
     if (
       typeof PDFJSDev === "undefined" ||
       PDFJSDev.test("!PRODUCTION || GENERIC")
     ) {
       if (
-        !(
-          this.container?.tagName.toUpperCase() === "DIV" &&
-          this.viewer?.tagName.toUpperCase() === "DIV"
-        )
+        this.container?.tagName.toUpperCase() !== "DIV" ||
```
Comment (calixteman) [inline web/pdf_viewer.js:240]: Unrelated to this patch, but could we remove the call to `toUpperCase` ?
According to https://developer.mozilla.org/en-US/docs/Web/API/Element/tagName#value, it should be useless in our case.

Comment (Snuffleupagus) [reply web/pdf_viewer.js:240]: I suppose the question is if we're, at all, worried about a third-party user trying to use the viewer in an XML document?
In that case this *might* matter, but probably not enough to worry about here (and this code won't run in MOZCENTRAL builds thanks to the pre-processor).

**Review by calixteman — APPROVED** [review]
LGTM.

**Pull request merged**